service/application/pom.xml

```diff
@@ -97,6 +97,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-validation</artifactId>
+        </dependency>
```

Comment:
So that we can use Constraint validations such as `@NotBlank` `@NotNull` etc.

Reply:
Example: https://github.com/SolaceProducts/event-management-agent/pull/185/files#diff-6bb24e9fb69000ce73f5944136797f3766f1c7a0146707771cc6466b3b757e0eR12-R20